src/main/resources/notification.yml

```
@@ -0,0 +1,3 @@
+notification:
+  discord:
+    webhook-url: ${discord-webhook-url}
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)